zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1272,6 +1272,44 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /elements/{elementKey}/variabels:
```

Comment:
I implemented straight forward my idea, this can be easily changed

Reply:
Looking at [the discussion](https://docs.google.com/document/d/1rS989EESojWRrCMLH7Nscy1UppbgOanvzVeOljCc_iA/edit?disco=AAABTUdE7BY) results, let's go for `element-instances` instead of `elements`. Wdyt?

Reply:
I'll change the path accordingly